test/integ/cli/test_cli_asset.py

```diff
@@ -0,0 +1,139 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+"""
+Integ tests for the CLI asset commands.
+"""
+import os
+import json
+from click.testing import CliRunner
+import pytest
+import tempfile
+import posixpath
+from deadline.job_attachments.asset_manifests.hash_algorithms import hash_file, HashAlgorithm
+
+from deadline.client.cli import main
+
+
+TEST_FILE_CONTENT = "test file content"
+TEST_SUB_DIR_FILE_CONTENT = "subdir file content"
+TEST_ROOT_DIR_FILE_CONTENT = "root file content"
+
+TEST_ROOT_FILE = "root_file.txt"
+TEST_SUB_FILE = "subdir_file.txt"
+
+TEST_ROOT_DIR = "root_dir"
+TEST_SUB_DIR_1 = "subdir1"
+TEST_SUB_DIR_2 = "subdir2"
+
+
+class TestSnapshot:
```

Comment:
I noticed that the unit tests and integration tests have the exact same test cases, except the unit tests use mocks. Could we tighten up the integration test cases a bit more to focus on testing actual integration points (e.g. reading/writing files)?

e.g. we probably do not needed the test cases here for when the command fails due to an invalid root directory, for example, since that is already covered by the unit tests.